plugins/org.jkiss.dbeaver.ext.generic/src/org/jkiss/dbeaver/ext/generic/model/TableCache.java

```diff
@@ -102,7 +102,7 @@ protected GenericTableColumn fetchChild(@NotNull JDBCSession session, @NotNull G
             log.warn("Error getting column scale", e);
         }
         Integer precision = null;
-        if (valueType == Types.NUMERIC || valueType == Types.DECIMAL) {
+        if (valueType == Types.NUMERIC || valueType == Types.DECIMAL || valueType == Types.BIGINT) {
```

Comment:
Not sure that we should use for all drivers.
BIGINT doesn't have precision (it is an integer).

Reply:
Ok. Logic moved.